meson.build

```diff
@@ -0,0 +1,130 @@
+project(
+    'dunst',
+    'c',
+    version: '1.9.2',
+    license: 'MIT',
+    meson_version: '>=0.60.0',
+    default_options: [
+        'c_std=gnu11',
```

Comment:
gnu99 seems to be enough.
also the only divergence from c99 that I can find is the use of variadic macros in our codebase with no extra args